terraform/applications/etherpad/state.tf

```diff
@@ -0,0 +1,26 @@
+terraform {
+  backend "s3" {
+    bucket = "itse-apps-stage-1-state"
+    key    = "us-west-2/etherpad/stage/terraform.tfstate"
+    region = "us-west-2"
+  }
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws",
+      version = "~> 3.0"
+    }
+  }
+
+  required_version = ">= 0.12"
```

Comment:
This should be 1.0.4, to match the tfenv version.

Reply:
done